core/src/commands/area.cpp

```diff
@@ -102,6 +102,8 @@ void AOClient::cmdUnCM(int argc, QStringList argv)
 
 void AOClient::cmdInvite(int argc, QStringList argv)
 {
+    Q_UNUSED(argc);
```

Comment:
This is fine for now, hopefully when I ever get around to the command/packet refactor these can be removed

Reply:
It is a temporary stopgap solution, yes.